release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -1385,6 +1385,7 @@ See link:https://access.redhat.com/articles/6955985[Navigating Kubernetes API de
 . When the vMedia attachment request fails with an error indicating that the `TransferProtocolType` attribute is missing, retry the request and explicitly specify this attribute.
 . Check for the presence of the RedFish settings resource in the system. If it is present, use it for the boot sequence override.
 
++
```

Comment:
Do you want the + to carry over the formatting of the numbered item preceeding ("Check for the presence..."). if so, I think you need to delete the blank line (1387) before the +. At the moment, no special formatting is carried over.

<img width="822" alt="image" src="https://user-images.githubusercontent.com/83772007/183483387-81bcad14-2dbc-42e5-8bd4-86b77acc7b12.png">

Reply:
I am going to adjust the wording of that last sentence. I want the formatting from the bullet carried over, not to the numbered items.